### ISSUE TITLE
PoseTrack Training

### ISSUE THREAD

**Issue by lxtGH** [mks0601/TF-SimpleHumanPose#45]
Hi! Thanks for your repo.
The number of joints in posetrack is a little different from COCO. How did you handle the missing joints like left_ear and right_ear. Also the different joint head_bottom and head_top?

**Comment by mks0601** [comment]
Just use pre-trained model on COCO, and set loss zero for not annotated l_ear and r_ear. When fine-tuning, slight different joint (head_bottom and head_top) can be handled by learning.

**Comment by lxtGH** [comment]
Hi! Thanks for your kind reply!!!!. During the prediction, we just ignore the channel for l_ear and r_ear? Have you tried to use 15 keypoints to finetune from COCO?  @mks0601

**Comment by lxtGH** [comment]
Also in the paper, they only train 20 epoch.  "During training, the base learning rate is 1e-4. It drops
to 1e-5 at 10 epochs and 1e-6 at 15 epochs. There are 20 epochs in total."  Why you train 150 epochs using lower lr ?

**Comment by mks0601** [comment]
I cannot understand your question. Could you make it clear?

**Comment by lxtGH** [comment]
@mks0601 Sorry for that. I update my moments.

**Comment by mks0601** [comment]
I think so, but cannot remember clearly. Sorry :(
